agent/src/beerocks/fronthaul_manager/ap_manager/ap_manager_thread.cpp

```diff
@@ -91,12 +91,13 @@ static void copy_vaps_info(std::shared_ptr<bwl::ap_wlan_hal> &ap_wlan_hal,
 using namespace beerocks;
 using namespace son;
 
-ap_manager_thread::ap_manager_thread(std::string slave_uds_)
-    : socket_thread(), bss_steer_valid_int(BSS_STEER_VALID_INT),
-      bss_steer_imminent_valid_int(BSS_STEER_IMMINENT_VALID_INT)
+ap_manager_thread::ap_manager_thread(const std::string &slave_uds_, const std::string &iface,
+                                     beerocks::logging &logger)
+    : socket_thread(), m_logger(logger)
 {
     thread_name = "ap_manager";
     slave_uds   = slave_uds_;
+    m_iface     = iface;
```

Comment:
If you add a member with m_, you're breaking the style in this file. I suggest to add a preparative commit which adds the _m to all members in this file first.

Reply:
I followed our coding style document. I can refactor this if you want.
But anyway it will be a separate commit at the end.
Please reply if you want from me to add a commit that refactors this.

Reply:
Although I absolutely applaud the concept of doing small refactorings like this as part of a feature-oriented PR, it is not a good idea to do that if the PR already has 38 commits... So no, please don't do that as part of this PR.

Reply:
Ok.